src/game/gameflow.c

```diff
@@ -1299,3 +1299,114 @@ GameFlow_InterpretSequence(int32_t level_num, GAMEFLOW_LEVEL_TYPE level_type)
 
     return ret;
 }
+
+GAMEFLOW_OPTION
+GameFlow_StorySoFar(
+    int32_t level_num, GAMEFLOW_LEVEL_TYPE level_type, int32_t savegame_level)
```

Comment:
I'm pretty sure `level_type` parameter is not needed here, there's no way to run this function on a saved game